tutorials/resources/build-golo/create-vm.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+vm_name="golo-lang-vm"
+vm_cpus=3
+vm_mem="4G"
+vm_disk="100GB"
+
+if [ ! -d "golo-distribution" ];then
+  mkdir golo-distribution
+fi
+
+multipass launch focal --name ${vm_name} --cpus ${vm_cpus} --mem ${vm_mem} --disk ${vm_disk} \
+  --cloud-init ./cloud-init.yaml
+
```

Comment:
maybe check if the VM exists and start it if required, to make the script idempotent.